src/prepack-cli.js

```diff
@@ -247,6 +247,9 @@ function run(
             1}) ${error.severity} ${error.errorCode}: ${error.message}` +
             ` (https://github.com/facebook/prepack/wiki/${error.errorCode})`
         );
+        if (foundFatal) {
+          console.error(error.callStack || "(Could not retrive callStack)");
```

Comment:
retrive -> retrieve.

But on the whole I would prefer not to see the message. The absence of a call stack is eloquent enough.